frontend/components/App/App.tsx

```diff
@@ -49,6 +52,10 @@ const App = ({ children, router }: IAppProps): JSX.Element => {
         setCurrentUser(user);
         setAvailableTeams(available_teams);
       } catch (error) {
+        console.error(error);
+        if (!location || location?.pathname === "/setup") {
+          return;
+        }
```

Comment:
You might be able to replace all of this with: 

`localStorage.removeItem("auth_token");`

And also get rid of the import. I tried locally and it's working for me. What do you think? 

That being said, this code does work, but my proposed solution would be less code unless you see any issues with it.

Reply:
@lukeheath do you mind dropping in your suggested edits that you have working locally? I just want to be sure I'm following your approach. Thanks!